src/loc/__snapshots__/ContextualizedLocDetails.test.tsx.snap

```diff
@@ -974,7 +974,7 @@ exports[`ContextualizedLocDetails renders for wallet user 1`] = `
                     <div
                       className="Col label"
                     >
-                      Legal Officer in charge
+                      Requested by
```

Comment:
I think that the test names ("renders for wallet user...") are not suitable anymore as `ContextualizedLocDetails` is for LOs only.